pkg/boot/common_settings.go

```diff
@@ -63,6 +64,9 @@ func (c *CommonSettings) Headers() http.Header {
 	if agent := headers.Get("User-Agent"); agent == "" {
 		headers.Set("User-Agent", "corectl")
 	}
+
+	headers.Set("Referer", c.GetContextName())
```

Comment:
As stated on the morning meeting, probably better to use the `--server` property instead.

Reply:
It sure is - thanks